Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-contests.tsx

```diff
@@ -291,7 +292,9 @@ const ContestsProvider = ({ children }: IContestsProviderProps) => {
                 filters,
                 sortingTypes,
                 contest,
-                isLoaded: isSuccess,
+                isLoaded: isSuccess, // should be removed, its incorrect logic, since isError is not taken into consideration
```

Comment:
Should be removed later or now?

Reply:
later, since it would require changes not relevant to the PR, the comment is generally addressing the issue that, isLoaded, should be based on that if you have retruned data, else is bad practice in my opinion